src/accelerate/commands/launch.py

```diff
@@ -307,6 +309,20 @@ def simple_launcher(args):
         mixed_precision = "fp16"
 
     current_env["MIXED_PRECISION"] = str(mixed_precision)
+    if args.num_cpu_threads_per_process == 1:
+        local_size = get_int_from_env(
+            ["MPI_LOCALNRANKS", "OMPI_COMM_WORLD_LOCAL_SIZE", "MV2_COMM_WORLD_LOCAL_SIZE"], 1
+        )
+        args.num_cpu_threads_per_process = int(psutil.cpu_count(logical=False) / local_size)
+        if args.num_cpu_threads_per_process == 0:
+            args.num_cpu_threads_per_process = 1
+        if args.num_cpu_threads_per_process != 1:
+            logger.warn(
+                'num_cpu_threads_per_process is reset to %d instead of default 1 to improve oob performance.\
+        \nyou could tun num_cpu_threads_per_process by pass --num_cpu_threads_per_process xxx to "accelerate launch "\n'
+                % args.num_cpu_threads_per_process
```

Comment:
Adapt the warning to the behavior suggested above (no default was provided, we picked xxx). Also use f-strings please.

Reply:
apply you suggestion @sgugger